CHANGES.md

```diff
@@ -6,6 +6,7 @@
 - Fix Migration stopping scheduled functions [#4704](https://github.com/appwrite/appwrite/pull/4704)
 - Fix Migration enabling OAuth providers with data by default [#4704](https://github.com/appwrite/appwrite/pull/4704)
 - Fix Error pages when OAuth providers are disabled [#4704](https://github.com/appwrite/appwrite/pull/4704)
+- Fix Locale API returning Unknown continent [#4761](https://github.com/appwrite/appwrite/pull/4761)
```

Comment:
Version 1.1.1 has already been released so probably should go in a new section, 1.1.2